packages/SwingSet/src/types-external.js

```diff
@@ -34,16 +34,11 @@ export {};
 /**
  * See ../docs/static-vats.md#vatpowers
  *
- * @typedef { MarshallingVatPowers & TerminationVatPowers } VatPowers
+ * @typedef { TerminationVatPowers } VatPowers
  *
  * @typedef { (VatPowers & MeteringVatPowers) } StaticVatPowers
  *
  * @typedef {{
- *   Remotable: unknown,
- *   getInterfaceOf: unknown,
- * }} MarshallingVatPowers
- *
- * @typedef {{
  *   makeGetMeter: unknown,
  *   transformMetering: unknown,
  * }} MeteringVatPowers
```

Comment:
Though obviously not for this PR, I am curious. Should we get rid of these too?

Reply:
yep, those are stale, I'll make a low-priority ticket to clean them up.